### ISSUE TITLE
`display_jpeg` and `display_png`: display size does not change on jupyter lab

### ISSUE THREAD

**Issue by kota7** [IRkernel/IRdisplay#50]
When I run the following code on Jupyter Notebook, the display size changes in accordance with the `width` option.

```r
library(ggplot2)
g <- qplot(1:5, 1:5)
ggsave("foo.jpeg", g, width=2, height=1)
IRdisplay::display_jpeg(file="foo.jpeg", width=100)
IRdisplay::display_jpeg(file="foo.jpeg", width=300)
```
But when I run the same code on Jupyter Lab, the display size does not change.
Same behavior for `display_png`.
Is this a bug or is there any fix to this?

I am using `IRkernel_1.0.2` and `IRdisplay_0.7.0`, and Python3.6 with `jupyter 1.0.0` and `jupytarlab 2.2.6` on a xubuntu 18.04 laptop.

On jupyter notebook:
![nb](https://user-images.githubusercontent.com/12293919/105046576-9c6a0280-5aac-11eb-823c-5a4d89f31222.png)

On jupyter lab:
![lab](https://user-images.githubusercontent.com/12293919/105046149-12219e80-5aac-11eb-9e40-62d0b841eb6a.png)

**Comment by kota7** [comment]
I found a workaround using `display_markdown`:

```r
library(ggplot2)
g <- qplot(1:5, 1:5)
ggsave("foo.jpeg", g, width=2, height=1)
IRdisplay::display_markdown('<img src="foo.jpeg" width="100">')
IRdisplay::display_markdown('<img src="foo.jpeg" width="300">')
```

It would be nice if `display_jpeg` and `display_png` also work, though.

**Comment by flying-sheep** [comment]
Oh! The fix has been in master for two years! 60a09b03f7a32413e6be166af219126198d41eb9

I just didn’t release it. No idea why, sorry. On it. Just install from git for the time being:

```r
devtools::install_github('IRkernel/IRdisplay')
```

**Comment by flying-sheep** [comment]
OK! 1.0 is released, and contains this.
